hexrd/ui/overlays/powder_overlay.py

```diff
@@ -384,8 +386,26 @@ def generate_ring_points(self, instr, tths, etas, panel, display_mode):
                          ang_crds[split_idx:, :]]
                     )
 
-                # append to list with nan padding
-                ring_pts.append(np.vstack([ang_crds, nans_row]))
+                if display_mode == ViewType.polar:
+                    # append to list with nan padding
+                    ring_pts.append(np.vstack([ang_crds, nans_row]))
+                elif display_mode == ViewType.stereo:
+                    # Swap back, convert to radians
+                    ang_crds[:, [0, 1]] = np.radians(ang_crds[:, [1, 0]])
+
+                    # Convert the ang_crds to stereo ij
+                    stereo_ij = angles_to_stereo(
+                        ang_crds,
+                        instr,
+                        HexrdConfig().stereo_size,
+                    )
+
+                    # FIXME: why??
+                    # swap i and j
+                    stereo_ij[:, [0, 1]] = stereo_ij[:, [1, 0]]
+
+                    # append to list with nan padding
+                    ring_pts.append(np.vstack([stereo_ij, nans_row]))
```

Comment:
@saransh13 Here is where for the powder overlays, we are taking the polar points and then projecting them into stereo coordinates.